textproc/libxmlxx3/Portfile

```diff
@@ -38,3 +40,6 @@ configure.args      --disable-silent-rules
 
 livecheck.type      gnome
 livecheck.name      ${gname}
+
+# Ignore major version updates.
+set livecheck.regex {LATEST-IS-(3\\.\\d*[02468](?:\\.\\d+)*)}
```

Comment:
I don't think the "set" should (or at least needs to) be here, or...?

Reply:
That `set` is what prevents this port from reporting libxml++ 5.01 as an update.

Reply:
I see... I understand that you changed the default regex to only pick up version 3. I just wasn't expecting you needed the `set` part here, as this is not added anywhere else in the tree - mostly just doing `livcheck.regex ???` is sufficient.
I can confirm though that without it, it doesn't behave as you want - I am just not sure why...